eng/Versions.props

```diff
@@ -2,10 +2,10 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.11.7</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
+    <VersionPrefix>17.11.9</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment:
Just curious - why we bump by two?

Reply:
Our pipeline or 17.11 was broken because there was a change in auth methods. To solve this we needed to merge a new Arcade update that bumped the version by 1 (https://github.com/dotnet/msbuild/pull/10654). Since this is the second PR that needs to go in we bumped another one. It will make more sense once the previous PR is merged in :)